docs/event-bus/docs/040-details-troubleshooting.md

```diff
@@ -0,0 +1,35 @@
+
+---
+title: Troubleshooting
+type: Details
+---
+## General Troubleshooting Guidelines
+
+* If the lambda or the service does not receive any Events, check the following:
+  - Confirm that the EventActivation is in place.
+  - Ensure that the webhook defined for the lambda or the service is up and running.
+  - Make sure the Events are published.
+
+* If errors appear while sending Events:
+  - Check if the `publish` application is up and running.
+  - Make sure that NATS Streaming is up and running.
+
+## Troubleshooting Using Kyma Tracing
+
+Tracing allows you to troubleshoot different problems that you might encounter while using Kyma. Understanding common scenarios and how the expected traces look gives you a better grasp on how to quickly use Kyma tracing capabilities to pinpoint the root cause.
```

Comment:
... `the common scenarios and how the expected traces look like` ...

Also: maybe it would be logical to add such a sentence:
`See the exemplary scenario for reference.`

... and then you start with the scenario.